onnx_tf/handlers/backend/det.py

```diff
@@ -0,0 +1,18 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_func
+
+
+@onnx_op("Det")
+@tf_func(tf.linalg.det)
+class Det(BackendHandler):
+
+  @classmethod
+  def version_11(cls, node, **kwargs):
+    tensor_dict = kwargs["tensor_dict"]
+    x = tensor_dict[node.inputs[0]]
+    return [
+        cls.make_tensor_from_onnx_node(node, inputs=[x])
```

Comment:
Just a nit. Could code be simplified as cls.make_tensor_from_onnx_node(node, **kwargs)?

Reply:
Thanks, updated.